scheduler/docker/docker.go

```diff
@@ -146,73 +146,75 @@ func NewScheduler(client *dockerutil.Client) *Scheduler {
 	}
 }
 
-func (s *Scheduler) Run(ctx context.Context, app *twelvefactor.Manifest, p *twelvefactor.Process, in io.Reader, out io.Writer) error {
+func (s *Scheduler) Run(ctx context.Context, app *twelvefactor.Manifest, in io.Reader, out io.Writer) error {
 	attached := out != nil || in != nil
 
 	if !attached {
 		return errors.New("cannot run detached processes with Docker scheduler")
 	}
 
-	labels := twelvefactor.Labels(app, p)
-	labels[runLabel] = Attached
-
-	if err := s.docker.PullImage(ctx, docker.PullImageOptions{
-		Registry:     p.Image.Registry,
-		Repository:   p.Image.Repository,
-		Tag:          p.Image.Tag,
-		OutputStream: replaceNL(out),
-	}); err != nil {
-		return fmt.Errorf("error pulling image: %v", err)
-	}
+	for _, p := range app.Processes {
```

Comment:
I kinda addressed this below, but the overall change is basically a noop, since Empire core only ever passes 1 process, so the end result is the same. It does mean that Empire core _could_ pass multiple processes to run them at once, but primarily this was done just to simplify the method signature since the manifest already contains all the process information.